impl/src/as/mod.rs

```diff
@@ -239,7 +239,9 @@ impl<'a> ToTokens for Expansion<'a> {
                 ImplKind::Specialized
             };
 
-            let trait_ty = quote! { ::core::convert::#trait_ident <#return_ty> };
+            let trait_ty = quote! {
+                ::derive_more::core::convert::#trait_ident <#return_ty>
```

Comment:
Can we use the shorter form here?
```suggestion
                ::derive_more::#trait_ident <#return_ty>
```

Reply:
@JelteF good catch 👍